src/button/default.scss

```
@@ -27,13 +36,34 @@
     }
     @include iui-notification-marker-hover;
 
-    > .iui-icon {
+    > .iui-button-icon {
       @include themed {
         fill: t(iui-icons-color-actionable-hover);
       }
     }
   }
 
+  &[disabled],
+  &[disabled]:hover,
+  &[disabled]:active,
+  &[disabled]:focus {
```

This selector doesn't need all these states. Also consider adding `&:disabled` because `[disabled] not work inside a fieldset. Need both because we are adding support for links as buttons, and `:disabled` will not work on links.

```suggestion
  &[disabled],
  &:disabled {
```